libs/shared/models/src/lib/event.model.ts

```diff
@@ -36,6 +36,7 @@ export interface IEvent {
   event_locations?: object;
   status?: string;
   collaboration_communities: ICommunity[];
```

Comment:
Why we add start_date in events, I think there is no column named "start_date" in events